main.go

```diff
@@ -27,15 +27,20 @@ type Record map[string]interface{}
 
 func TransformExportJsonRecord(wh warehouse.Warehouse, rec map[string]interface{}) ([]string, error) {
 	var line []string
-	for _, field := range warehouse.ExportTableSchema {
-		if val, ok := rec[field.Name()]; ok {
+	for _, field := range wh.ExportTableSchema() {
+		if field.IsCustomVars {
+			continue
+		}
```

Comment:
It turns out that having the CustomVars separate from the main schema wasn't quite vestigial; it was necessary to make the CustomVars JSON generated in this function work.  I wanted to handle the JSON here rather than each warehouse impl, so I added a new field to the Field struct to indicate whether the field is the designated "CustomVars" field.